exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter_options.h

```diff
@@ -15,38 +14,16 @@ namespace otlp
 
 /**
  * Struct to hold OTLP GRPC traces exporter options.
+ *
+ * See
+ * https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md#otlpgrpc
+ *
+ * See
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md
  */
-struct OtlpGrpcExporterOptions
+struct OtlpGrpcExporterOptions : public OtlpGrpcClientOptions
 {
-  // The endpoint to export to. By default the OpenTelemetry Collector's default endpoint.
-  std::string endpoint = GetOtlpDefaultGrpcEndpoint();
-  // By default when false, uses grpc::InsecureChannelCredentials(); If true,
-  // uses ssl_credentials_cacert_path if non-empty, else uses ssl_credentials_cacert_as_string
-  bool use_ssl_credentials = GetOtlpDefaultIsSslEnable();
-  // ssl_credentials_cacert_path specifies path to .pem file to be used for SSL encryption.
-  std::string ssl_credentials_cacert_path = GetOtlpDefaultSslCertificatePath();
-  // ssl_credentials_cacert_as_string in-memory string representation of .pem file to be used for
-  // SSL encryption.
-  std::string ssl_credentials_cacert_as_string = GetOtlpDefaultSslCertificateString();
-
-#ifdef ENABLE_OTLP_GRPC_SSL_MTLS_PREVIEW
-  // At most one of ssl_client_key_* should be non-empty. If use_ssl_credentials, they will
-  // be read to allow for mTLS.
-  std::string ssl_client_key_path   = GetOtlpDefaultTracesSslClientKeyPath();
-  std::string ssl_client_key_string = GetOtlpDefaultTracesSslClientKeyString();
-
-  // At most one of ssl_client_cert_* should be non-empty. If use_ssl_credentials, they will
-  // be read to allow for mTLS.
-  std::string ssl_client_cert_path   = GetOtlpDefaultTracesSslClientCertificatePath();
-  std::string ssl_client_cert_string = GetOtlpDefaultTracesSslClientCertificateString();
-#endif
-
-  // Timeout for grpc deadline
-  std::chrono::system_clock::duration timeout = GetOtlpDefaultTimeout();
-  // Additional HTTP headers
-  OtlpHeaders metadata = GetOtlpDefaultHeaders();
-  // User agent
-  std::string user_agent = GetOtlpDefaultUserAgent();
+  OtlpGrpcExporterOptions();
```

Comment:
Should we export constructor with `OPENTELEMETRY_EXPORT` if it's not a head-only implement?

Reply:
Yes, this was listed in the "Follow up work" in the PR comments.

I added the `OPENTELEMETRY_EXPORT` in the header files, so this part is done.

I do not know however how to test and adjust file `./ext/src/dll/opentelemetry_cpp.src`, having no access to a windows machine.

Reply:
Is there any test to cover this for DLL linking? Like trigger a link error if miss addming `OPENTELEMETRY_EXPORT`.